deploy/roles/web/handlers/main.yaml

```diff
@@ -1,11 +1,18 @@
 ---
 
+# gzip files for nginx's gzip_static
+- name: gzip files
+  command: "{{ item }}"
+  loop:
+    - "find /var/www/playreading/public -name '*.gz' -exec rm {} ;"
+    - "find /var/www/playreading/public -type f ( -name '*.css' -o -name '*.js' -o -name '*.html' ) -exec gzip --best --keep {} ;"
```

Comment:
`find`'s `-exec` argument runs a command for each file found.  
This is used to delete all the existing gz files for the first line and then compress each css/js/html file for the second line.

Reply:
Does this need to be in a loop?